main.go

```diff
@@ -18,6 +18,12 @@ var (
 )
 
 func main() {
+
+	err := logging.SetLogLevel("main", "INFO")
```

Comment:
Is this appropriate? would it be better to name this logger something other than "main"?

Reply:
I think that's OK. Becomes pretty clear from where this log is coming.

Reply:
Maybe `devnet-main`? Both LGTM.